internal/pkg/cli/task_run.go

```diff
@@ -31,11 +31,14 @@ var (
 )
 
 var (
+	fmtTaskRunAppPrompt = fmt.Sprintf("In which %s would you like to run this %s?", color.Emphasize("application"), color.Emphasize("task"))
 	fmtTaskRunEnvPrompt       = fmt.Sprintf("In which %s would you like to run this %s?", color.Emphasize("environment"), color.Emphasize("task"))
```

Comment:
oohh also, we only prefix the string with `fmt` if the string contains format verbs that needs to be rendered at runtime. In this situation the string is already rendered so the variable names should be `taskRunAppPrompt` and `taskRunEnvPrompt`